webview/src/getStarted/components/ProjectUninitialized.tsx

```diff
@@ -0,0 +1,28 @@
+import { VSCodeButton } from '@vscode/webview-ui-toolkit/react'
+import React from 'react'
+import { EmptyState } from '../../shared/components/emptyState/EmptyState'
+
+interface ProjectUninitializedProps {
+  initializeProject: () => void
+}
+
+export const ProjectUninitialized: React.FC<ProjectUninitializedProps> = ({
+  initializeProject
+}) => {
+  return (
+    <EmptyState>
+      <div>
+        <h1>DVC is not initialized</h1>
+        <p>
+          The currently open folder does not contain a DVC project. You can
```

Comment:
[C] Could swap `The currently open folder` for `The current workspace`

Reply:
Will change it in both places (sidebar and webview)